src/keyboard/system76/14in_83/include/board/keymap.h

```diff
@@ -79,4 +79,26 @@
 #define MATRIX_FN_INPUT     0
 #define MATRIX_FN_OUTPUT    6
 
+static inline bool keymap_is_f_key(uint8_t row, uint8_t col) {
```

Comment:
This function may not be valid on the combo_iso_ansi layout

Reply:
All `K0*` keys are mapped to the same slots for both layouts, looking at the two macros above, since both layouts have `fn` keys in the same places. So i think it should be fine.

Reply:
Ok, thanks for checking